### ISSUE TITLE
Ray + Pytorch distributed?

### ISSUE THREAD

**Issue by kellyaa** [caikit/caikit-ray-backend#28]
How can Ray work with caikit modules that are using pytorch for internal distribution?

**Comment by kellyaa** [comment]
Initial hypothesis is that since the distribution being used by caikit-nlp fine tuning does not reach outside of a single node, this should not become an issue or contend with Ray resource management. I'm going to keep this open, however, and close it only after this hypothesis has been tested.

**Comment by kellyaa** [comment]
Declaring this out of scope for this module.
